TestCases/nicf/coolprop/coolprop.cfg

```diff
@@ -0,0 +1,238 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Supersonic flow over a wedge in a channel.                 %
+% Author: Thomas D. Economon                                                   %
+% Institution: Stanford University                                             %
+% Date: 2012.10.07                                                             %
+% File Version 5.0.0 "Raven"                                                   %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= RANS
+KIND_TURB_MODEL= SA
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
+MATH_PROBLEM= DIRECT
+% Restart solution (NO, YES)
+RESTART_SOL= NO
+
```

Comment:
Please try to extrapolate from comments on previous PR's, it is tiresome to repeat the same thing all the time... update the header, remove comments from the file, especially lists of options that do not exist anymore...

Reply:
Thank you, I will follow this rule...